data/tooltips.txt

```diff
@@ -98,6 +104,42 @@ tip "cloaking fuel:"
 tip "cloaking heat:"
 	`Heat generated per second while cloaked.`
 
+tip "cloaking shield:"
+	`Shields consumed per second while cloaked.`
+
+tip "cloaking shield delay:"
+	`This ship is unable to regenerate shields whilst cloaked, instead its shield regeneration will suffer a delay that will grow the longer it is cloaked`
+
+tip "cloaking hull:"
+	`Hull consumed per second while cloaked.`
+
+tip "cloaking hull delay:"
+	`This ship is unable to regenerate hull whilst cloaked, instead its hull regeneration will suffer a delay that will grow the longer it is cloaked.`
+
+tip "cloaking hull protection:"
+	`Protection against weapons that do hull damage when the cloak is active. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in hull damage, while a total value of 11 only results in a 10 percent reduction.`
+
+tip "cloaking shield protection:"
+	`Protection against weapons that do shield damage when the cloak is active. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
+
+tip "cloaking phasing"
+	`Weapons cannot affect this ship, and pass right trough.`
+
+tip "cloaking visibility:"
+	`This cloaking device has flaws, and as a result you can still be partially seen by enemies. This number represents the maximum amount of cloaking you can reach; up to 1. It has an effect on optical tracking, making your ship easier to spot by those missiles the more visible you are.`
+
+tip "cloaking targetability:"
+	`This value represents how easy this ship is to track using missiles and other guidance systems, when the cloak is active. All missiles will have a really hard time locking on you if it is low enough, but ships will still be able to keep a lock on you anyway.`
+
+tip "cloaking show on radar:"
+	`This cloaking device does not hide you from radars. On top of that, it has an effect on radar trackers. The lower this value is, the easier time tracking you they will have.`
+
+tip "cloaking action:"
+	`This device is so efficient at hiding you that you may shoot when cloaking, losing this amount of cloaking in the process, before being cloaked again. You can also board and hail when you are cloaked, using this device. A negative value means shooting does not even hinder your cloak.`
```

Comment:
I don't agree on that, I think it makes more sense to define how much cloak is removed per shot, otherwise it gets really weird.
We could have a new attribute that would do that though ~~but I try not to add too many, I guess I'll just have to~~